cli/update-version.js

```diff
@@ -1,8 +1,8 @@
-const chalk = require('chalk'),
-	git = require('simple-git/promise')(),
+const git = require('simple-git/promise')(),
```

Comment:
Removed chalk since we don't run this locally and Travis doesn't show the colours anyway.